vcr/revocation/statuslist2021_issuer.go

```diff
@@ -426,7 +426,6 @@ func (cs *StatusList2021) Revoke(ctx context.Context, credentialID ssi.URI, entr
 		}
 
 		// append new revocation and re-issue the StatusList2021Credential.
-		issuerRecord.Revocations = append(issuerRecord.Revocations)
```

Comment:
isn't this an actual bug, since there's nothing appended?

Reply:
This used to append the revocation from L397, but the query on L414 now includes new revocation. I just forgot to remove this. The expected behavior is tested.